docs/source/impl-guide/normalization.rst

```diff
@@ -88,38 +113,33 @@ alternate sequence) at an interval is outlined below.
    :widths: 40 15 20
    :align: left
 
-   *  -  Steps
-      -  | Interbase Position
-         | and Alleles
-      -  | Resulting Allele Set
-         | (All alleles in this column result
-	 | in the same empirical sequence change.)
+   *  -  | Steps
+      -  | `start` and `end` (interbase)
+	 | and allele sequences
+      -  | Equivalent representations
    *  -  0. Given allele ``S:g.5_6delinsCAGCA`` defined on reference sequence S=TCAGCAGCT
       -  | (4,6)
          | (“CA”, “CAGCA”)
       -  .. math:: TCAG \Bigl[ \frac{CA}{CAGCA} \Bigr] GCT
 
    *  -  1. Trimming
 
-            Remove prefix common to all alleles, if any, and update start position. Remove suffix common to all alleles, if any, and update end position.
+            a. Remove suffix common to all alleles, if any, and update end position.
+            b. Remove prefix common to all alleles, if any, and update start position. 
 
             **Note:**  This example shows removing C prefix and A suffix.
             Equivalently in this case, CA prefix or CA suffix could be removed.
       -  | (5,5)
          | (“”, “AGC”)
-      -  .. math:: TCAGC \Bigl[ \frac{}{AGC} \Bigr] AGCT  ①
-   *  -  2. Condition: One allele must be empty.
+      -  .. math:: TCAGC \Bigl[ \frac{}{AGC} \Bigr] AGCT
 
-            If the reference allele is empty, the allele set represents an insertion in the reference.
-
-            If the alternate allele is empty, the allele set represents a deletion in the reference.
-
-            If neither is true, the allele set represents a substitution, which is not subject to further normalization.
+   *  -  2. & 3. These conditions are False.
```

Comment:
Formatting for readability: I am scrolling back and forth between the table example. Is the table wide enough to include the descriptions of each normalization step?How about a link out for a side-by-side comparison of process vs. example?

Reply:
I agree that the web page constraints require flipping back and forth and that it's awkward. I'll spend a small bit of time trying to work out something better.

Reply:
I don't see an easy way to generate a more concise/readable table with sphinx. PRs or alternative suggestions are welcomed, but I'm going to resolve this for now since it's not significant enough to block merge.